types/src/system/auction/providers.rs

```diff
@@ -52,8 +52,8 @@ pub trait StorageProvider {
         unbonding_purses: Vec<UnbondingPurse>,
     ) -> Result<(), Error>;
 
-    /// Records era info at the given era id.
-    fn record_era_info(&mut self, era_id: EraId, era_info: EraInfo) -> Result<(), Error>;
+    /// Records era summary.
+    fn record_era_summary(&mut self, _era_id: EraId, era_summary: EraInfo) -> Result<(), Error>;
```

Comment:
As per the previous comment on this method, this is still technically a breaking change as we've renamed the method.

Reply:
Renamed back to `record_era_info`